stl/inc/format

```diff
@@ -1991,6 +1991,9 @@ private:
     const _Format_arg_index* _Index_array = nullptr;
 };
 
+template <class _Context, class... _Args>
+basic_format_args(_Format_arg_store<_Context, _Args...>) -> basic_format_args<_Context>;

```

Comment:
No change requested, note to other reviewers: Deduction guides can't be exported for modules, they're just automatically available if the class is exported (which is the case here), so the lack of any `_EXPORT_STD` is normal.